### PR TITLE
NMS-10477 : Use a utility function instead of Integer.getInteger

### PR DESCRIPTION
**Pull request by patrick-schweizer**
* JIRA: https://issues.opennms.org/browse/NMS-10477

Our [continuous integration system](http://bamboo.internal.opennms.com:8085) will test and verify your changes.

### CODE REVIEW

**Comment by roskens** [comment]
This is pretty neat to see.

In reviewing some of the changes, I see there were inconsistent coding patterns prior to these changes.

* TimeUnit.<period> usage
* Millisecond vs Seconds: where the property does not specify the expected period its name.

Would this class allow for use of <num><period> (ex: 1s, 5m) as property values in the future?

**Comment by j-white** [comment]
(You'll need to merge/rebase with `develop` to get those failing integration tests to pass.)

**Comment by patrick-schweizer** [comment]
> (You'll need to merge/rebase with `develop` to get those failing integration tests to pass.)
Thanks @j-white will do.

**Comment by mvrueden** [comment]
Added `triage` label as there is a `TODO` in the code.

File: core/soa/src/main/java/org/opennms/core/soa/lookup/ServiceLookupBuilder.java [hunk 1]
```diff
@@ -80,4 +88,22 @@ public ServiceLookupBuilder blocking(long gracePeriodInMs, long sleepTimeInMs, l
         return serviceProvider;
     }
 
+    /** TODO Patrick: needs to be replaced by SystemProperties */
```
[the remark this answers is not in the document]
Comment (patrick-schweizer) [reply core/soa/src/main/java/org/opennms/core/soa/lookup/ServiceLookupBuilder.java]: @j-white ?

[the remark this answers is not in the document]
Comment (j-white) [reply core/soa/src/main/java/org/opennms/core/soa/lookup/ServiceLookupBuilder.java]: I'd go for option 2. with a new module like `core/sysprops/pom.xml`.

[the remark this answers is not in the document]
Comment (patrick-schweizer) [reply core/soa/src/main/java/org/opennms/core/soa/lookup/ServiceLookupBuilder.java]: ok, will do

**Comment by patrick-schweizer** [comment]
remove triage => TODO is fixed

**Pull request merged**